man/GGIR.Rd

```diff
@@ -1124,12 +1138,12 @@ GGIR(mode = 1:5,
         below 20 per cent of its standard deviation.}
     
       \item{HDCZA_threshold}{
-        Numeric (default = 0.2)
-        If \code{HASPT.algo} is set to "HDCZA" then HDCZA_threshold will be used
-        as threshold instead of 6th step in the diagram of Figure 1 in
-        van Hees et al. 2018 Scientific Report (doi: 10.1038/s41598-018-31266-z). We have
-        now simplified this step to a constant number, which can be modified via 
-        HDCZA_threshold.
+        Numeric (default = c())
+        If \code{HASPT.algo} is set to "HDCZA" and HDCZA_threshold is not NULL then
+        it will be used as threshold instead of 6th step in the diagram of Figure 1 in
+        van Hees et al. 2018 Scientific Report (doi: 10.1038/s41598-018-31266-z). This
```

Comment:
Thanks, I will rephrase. But discussing the 6th step seems hard: The threshold in the 6th step is the 10th percentile of the values produced by step 5 multiplied by 15. I am not sure this will provide much clarify because then we also need to explain what step 5 does, which requires a full discussion of the entire algorithm.